pkg/rlptools/wasm_utils.go

```diff
@@ -30,7 +31,16 @@ func WasmRules(rlp *kuadrantv1beta2.RateLimitPolicy, route *gatewayapiv1.HTTPRou
 		return rules
 	}
 
-	for limitName := range rlp.Spec.Limits {
+	// Sort RLP limits for consistent comparison with existing wasmplugin objects
+	limitNames := make([]string, 0, len(rlp.Spec.Limits))
```

Comment:
This mitigates but does not solve 100%, I'm afraid. _Ceteris paribus_ 🤓, a consistent wasmplugin config also depends on the order of the policies which are indexed for a gateway using a regular Golang map.

Reply:
The order of the policies is already being sorted [here](https://github.com/Kuadrant/kuadrant-operator/blob/a46f05f93176c90455d174476d13aa059d0fcde2/controllers/rate_limiting_wasmplugin_controller.go#L173), isn't it?

Reply:
as I mentioned already
> The controller [sorts that list](https://github.com/Kuadrant/kuadrant-operator/blob/wasmplugin-controller/controllers/rate_limiting_wasmplugin_controller.go#L172-L173) of policies afterwards based on policy creation timestamp